taglibs/async/await-tag.js

```diff
@@ -23,6 +23,10 @@ function promiseToCallback(promise, callback, thisObj) {
             finalPromise = finalPromise.fail(function(err) {
                 callback(err);
             });
+        } else {
+            finalPromise = finalPromise.then(void 0, function(err) {
```

Comment:
The point of using `void 0` is that it *always* produces a literal `undefined` (ever since the very first version of javascript), while the `undefined` variable can be redefined.